internal/c/libqb/include/compression.h

```diff
@@ -9,5 +9,7 @@
 
 struct qbs;
 
+uint32_t func__adler32(qbs *text);
```

Comment:
Rather than put them in `font.h` and `compression.h` I think it would make more sense to add a new `hashing.h` and put them in there.